connectors/salesforce/src/main/java/com/vmware/connectors/salesforce/SalesforceController.java

```diff
@@ -45,16 +45,16 @@
 
 @RestController
 public class SalesforceController {
-    private final static Logger logger = LoggerFactory.getLogger(SalesforceController.class);
-    private final static String SALESFORCE_AUTH_HEADER = "x-salesforce-authorization";
-    private final static String SALESFORCE_BASE_URL_HEADER = "x-salesforce-base-url";
-    private final static String ROUTING_PREFIX = "x-routing-prefix";
+
+    private static final Logger logger = LoggerFactory.getLogger(SalesforceController.class);
+
+    private static final String SALESFORCE_AUTH_HEADER = "x-salesforce-authorization";
+    private static final String SALESFORCE_BASE_URL_HEADER = "x-salesforce-base-url";
+    private static final String ROUTING_PREFIX = "x-routing-prefix";
     private static final String ADD_CONVERSATIONS_PATH = "/conversations";
     private static final String CONVERSATION_TYPE = "email";
 
-    private final static String WHITE_SPACE = " ";
-    private final static String NEW_LINE = "\n";
-    private final static int SIZE = 256;
+    // TODO: concatenating strings into a SOQL query like this may provide an avenue for a SOQL-injection attack
```

Comment:
This comment used to be down by the 3 parallel calls (when they use String.format to construct the SOQL).  Now that I moved those around to different places, this made more sense to be near the constants that defined the format string.